### ISSUE TITLE
enable pre-EIP-155 tx type

### ISSUE THREAD

**Issue by brewmaster012** [zeta-chain/node#1359]
Some uses cases require deploying contract to the same address on multiple networks. 
Examples include this  proxy contract: https://github.com/Arachnid/deterministic-deployment-proxy

The above repo took the approach of using a pre-EIP-155 transaction to deploy the contract. The idea
is that pre-EIP-155 transaction does not include chainID therefore will deploy to the same contract address
on all networks. 

Right now pre-EIP-155 transaction types are blocked by zetachain EVM. We should consider enabling it. 
https://github.com/evmos/ethermint/blob/eb3cc87c9b24fa7f63c156ad6fb1d071a6f80176/proto/ethermint/evm/v1/evm.proto#L23

**Comment by lumtis** [comment]
Closing as it seems no longer relevant with the current state of the project. Feel free to reopen it.
A PR has recently been merged to fix the config reading
